### ISSUE TITLE
Message geenration with rusp

### ISSUE THREAD

**Issue by mzribi** [axiros/rusp#2]
Hellio,
I have this problem when trying to generate msg with rusp encode_msg or encode_msg_body

$ rusp encode_msg_body -f file.probuff Get
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("EOF while parsing a value", line: 1, column: 0)', src/libcore/result.rs:1009:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

**Comment by therealprof** [comment]
You need to specify the paths of what you'd like to `Get` after the command, e.g.
```
$ rusp encode_msg_body -f file.probuff Get '["Device."]'
```

**Comment by therealprof** [comment]
Please feel free to send a PR with documentation or usability improvements.

**Comment by mzribi** [comment]
Thank youn very much.

**Comment by mzribi** [comment]
According to the documentation we can use only 
    Get        Generate an USP Get request message
    GetResp    Generate an USP GetResp response message
    
Can we use other message type like :

Get get
GetSupportedDM get_supported_dm
GetInstances get_instances
Set set
Add add
Delete delete
Operate operate
Notify notify
GetSupportedProtocol get_supported_protocol

**Comment by mzribi** [comment]
Can you give us how to use rusp in the other case ?

**Comment by therealprof** [comment]
Yes, whenever I get around to implementing them or someone sends a PR to implement it.
